src/main/xerus/monstercat/tabs/TabPlaylist.kt

```diff
@@ -0,0 +1,103 @@
+package xerus.monstercat.tabs
+
+import javafx.beans.Observable
+import javafx.beans.value.ObservableValue
+import javafx.scene.control.*
+import javafx.scene.input.KeyCode
+import javafx.scene.input.MouseButton
+import javafx.util.Callback
+import xerus.ktutil.javafx.MenuItem
+import xerus.ktutil.javafx.fill
+import xerus.ktutil.javafx.properties.ImmutableObservable
+import xerus.ktutil.javafx.properties.listen
+import xerus.monstercat.api.Player
+import xerus.monstercat.api.Playlist
+import xerus.monstercat.api.response.Track
+
+
+class TabPlaylist : VTab() {
+	var table = TableView<Track>().apply {
+		columnResizePolicy = TableView.CONSTRAINED_RESIZE_POLICY
+	}
+	
+	init {
+		table.items = Playlist.tracks
+		
+		table.columns.addAll(TableColumn<Track, String>("Artists").apply {
+			cellValueFactory = Callback<TableColumn.CellDataFeatures<Track, String>, ObservableValue<String>> { p ->
+				ImmutableObservable(p.value.artistsTitle)
+			}
+		}, TableColumn<Track, String>("Title").apply {
+			cellValueFactory = Callback<TableColumn.CellDataFeatures<Track, String>, ObservableValue<String>> { p ->
+				ImmutableObservable(p.value.title)
+			}
+		})
+		
+		table.setRowFactory {
+			TableRow<Track>().apply {
+				Playlist.currentIndex.addListener { _, _, newValue ->
+					style = if (index == newValue) {
+						"-fx-background-color: #1f6601"
```

Comment:
Please put the if-else into the string so that you don't duplicate the common string

Reply:
844c9bdc5333ba1fa02db6d856be0906c3f13ee1

Reply:
844c9bdc5333ba1fa02db6d856be0906c3f13ee1